YourEasyRent/Services/TelegramBotServices.cs

```diff
@@ -6,6 +6,7 @@
 using System;
 using Telegram.Bot.Args;
 using Telegram.Bot.Types.ReplyMarkups;
+using Microsoft.VisualBasic;
```

Comment:
Wtf is this?

Reply:
i dont know

Reply:
Try to delete it and see what breaks, it looks like it should not be needed.